template/package.json

```diff
@@ -16,6 +16,15 @@
     "prepare": "husky install",
     "postinstall": "bash ./scripts/post-install.sh",
     "reverse": "adb reverse tcp:8081 tcp:8081",
+    "android:stg": "react-native run-android --variant=stagingDebug",
+    "android:stg-release": "react-native run-android --variant=stagingRelease",
+    "android:dev": "react-native run-android --variant=developmentDebug",
+    "android:dev-release": "react-native run-android --variant=developmentRelease",
+    "android:prod": "react-native run-android --variant=productionDebug",
+    "android:prod-release": "react-native run-android --variant=productionRelease",
+    "ios:prod": "react-native run-ios --scheme 'RNBaseProject'",
+    "ios:dev": "react-native run-ios --scheme 'RNBaseProjectDev'",
+    "ios:staging": "react-native run-ios --scheme 'RNBaseProjectStg'",
```

Comment:
Please create two scripts and add a function to pass variant (android), scheme (ios)

Reply:
I've updated it, please review it again